pkg/utils/quantity.go

```diff
@@ -45,6 +45,20 @@ func TransformQuantityToJindoUnit(q *resource.Quantity) (value string) {
 	return
 }
 
+// TransformQuantityToGooseFSUnit transform a given input quantity to another one
+// that can be recognized by GooseFS. This is necessary because GooseFS takes decimal byte units(e.g. KB, MB, GB, etc.)
+// as binary byte units(e.g. Ki, Mi, Gi)
+func TransformQuantityToGooseFSUnit(q *resource.Quantity) (value string) {
+	value = q.String()
+
+	if strings.HasSuffix(value, "i") {
+		value = strings.ReplaceAll(value, "i", "B")
```

Comment:
this logic `strings.ReplaceAll` means that  1Ki == 1KB is true ?

Reply:
right, Ki is legal for kubernetes and KB is legal for Alluxio and GooseFS.